Library/Homebrew/test/rubocops/formula_desc_cop_spec.rb

```diff
@@ -162,6 +162,26 @@ class Foo < Formula
       end
     end
 
+    when "desc ends in a full stop" do
```

Comment:
Thanks for pointing this out. I should (and do) know this (and I'm frustrated with myself)? 💦 

However, every other `it "When ...` block in this file is the same, there are no `context`s. I'm loathe to change them all in this PR, but I agree that "When" should be a `context`. What should I do?